cpp/ps0000/p0000-soln-cpp.cpp

```diff
@@ -0,0 +1,7 @@
+#include <iostream> // 2/2
+int main(void) // 1/1
+{
+  std::cout << "Hello, World!\n";  // 4/4

```

Comment:
- [x] used `<<` for outputting instead of `>>`
- [x] used `\n` or `std::endl` in the right way
- [x] avoided syntax error by typing `;` at the end of the line
- [x] output is as required `Hello, World!` in a single line